server/spec/auth/authenticate_user_spec.rb

```diff
@@ -0,0 +1,33 @@
+# spec/auth/authenticate_user_spec.rb
+require 'rails_helper'
+
+RSpec.describe AuthenticateUser do
+  # create test user
+  let(:user) { create(:api_user) }
+  # valid request subject
+  subject(:valid_auth_obj) { described_class.new(user.email, user.password) }
```

Comment:
Naming: the valid_auth_obj is a abbreviated name. If we had a longer, more descriptive name, we could drop the code comment about it.

Reply:
updated name